src/System.Memory/tests/ReadOnlySpan/IsWhiteSpace.cs

```diff
@@ -30,13 +30,13 @@ public static void IsWhiteSpaceTrueLatin1()
                     a[i] = s_whiteSpaceCharacters[rand.Next(0, s_whiteSpaceCharacters.Length - 1)];
                 }
                 var span = new Span<char>(a);
-                bool result = span.AsReadOnlySpan().IsWhiteSpace();
+                bool result = ((ReadOnlySpan<char>)span).IsWhiteSpace();
```

Comment:
I assume we have IsWhitespace that extends Span<char> directly so people don't have to do this cast. i.e. the cast is for testing purposes only.

Reply:
IsWhiteSpace extends `ReadOnlySpan<char>` only and is not visible on `Span<char>`. So, if someone has `Span<char>`, they would need the explicit cast or call the ReadOnlySpan ctor (or rely on a local ReadOnlySpan from the span, using implicit cast).